Collector Version 19.456-3 release note (#2942)
SumoLogic/sumologic-documentation: blog-collector/2023-08-21.md

---
title: Version 19.456-3
image: https://help.sumologic.com/img/sumo-square.png
authors:
- url: https://help.sumologic.com/release-notes-collector/rss.xml
  image_url: /img/release-notes/rss-orange.png
---

> In this release, we've enhanced the security and stability of Collector with added support for security patch and bug fix.

Reviewer: ```suggestion
In this release, we've enhanced our Collector security and stability by adding support for security patches and bug fixes.
```
